vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporterController.java

```diff
@@ -180,9 +180,6 @@ private MongoCredentials getCredentials(String species, String dbName,
             }
         }
 
-        if (dbName == null) {
```

Comment:
dbName is not null, both in the CLI and WS versions of the dumper, so removing those lines is good